sequence.go

```diff
@@ -36,6 +51,18 @@ func (s *Sequence) Next(state State) uint64 {
 	}
 
 	state.Set(s.Key, buf.Bytes())
+	return seq
+}
+
+func (s *Sequence) Next2(kvStore store.KVStore) uint64 {
```

Comment:
And the `State` already implements `KVStore`, so this function is also redundant.

Reply:
Remove `Next2`